src/zarate.jl

```diff
@@ -52,7 +54,8 @@ for e in edges(g)
 end
 ```
 """
-function solve_positions(layout::Zarate, original_graph)
+function solve_positions(layout::Zarate, original_graph;
+    opt_node_order::Dict{Int, Int} = Dict{Int, Int}())
```

Comment:
Rather than a dictionary, I think we should probably use a Vector of Pairs.
Because one might want to specify multiple items that much occur above
e.g `[1 => 2,  1=>3, 2=>3]`

Alternatively, perhaps we should make it be a vector of integers, and you just list everything in order that it should occur, and anything that doesn't occur is allowed anywhere.
That would force more constraints though and be less flexible, but it might be more initutive?
Probably more constraints would be bad though, we rely on the flexibility of being able to reorder things.

Reply:
I agree, the current formulation should also work without it being explicitly a Dict